docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md

```diff
@@ -2,4 +2,96 @@
 title: ✳ How to create and edit Expectations in bulk
 ---
 
-This article is a stub.
+The `JsonSchemaProfiler` helps you quickly create [Expectation Suites](/docs/reference/expectation-suite-operations) from jsonschema files.
+
+:::note Prerequisites 
+This how-to guide assumes you have already:
+
+* Set up a [working deployment of Great Expectations](/docs/tutorials/getting-started/intro)
+* Have a valid jsonschema file that has top level object of type object.
+
+:::
+
+:::warning
+
+This implementation does not traverse any levels of nesting.
+:::
+
+### Steps
+
+#### 1.Set a filename and a suite name
+
+````console
+jsonschema_file = "YOUR_JSON_SCHEMA_FILE.json"
+suite_name = "YOUR_SUITE_NAME"
+````
+
+#### 2. Load a DataContext
+
+````console
+context = ge.data_context.DataContext()
+````
+
+#### 3. Load the jsonschema file
+
+````python
+with open(jsonschema_file, "r") as f:
+    schema = json.load(f)
+````
+
+#### 4. Instantiate the profiler
+
+````python
+profiler = JsonSchemaProfiler()
+````
+
+#### 5. Create the suite
+
+````python
+suite = profiler.profile(schema, suite_name)
+````
+#### 6. Save the suite
+
+````python
+context.save_expectation_suite(suite)
+````
+
+#### 7. Optionally, generate Data Docs and review the results there.
+
+Data Docs provides a concise and useful way to review the Expectation Suite that has been created.
+
+````console
+   context.build_data_docs()
+
+You can also review and update the Expectations created by the profiler to get to the Expectation Suite you want using ``great_expectations suite edit``.
+````
```

Comment:
Please left-align `context.build_data_docs()` so it matches the rest of the code blocks. Additionally, the following sentence shouldn't be wrapped in the code block right?